themes/openy_themes/openy_rose/scss/modules/_location.scss

```diff
@@ -0,0 +1,20 @@
+.paragraph--type--prgf-location-finder {
+  .views-element-container {
+    padding: 0;
+    .locations-list {
+      padding: 20px 0;
+      .label-for-field-location-fax,
+      .label-for-field-location-phone {
+        text-transform: lowercase;
+        display: inline;
+      }
+      .field-location-fax,
+      .field-location-phone {
+        display: inline;
+      }
+      h1 {
```

Comment:
I guess that was force color for word "YMCAs" when it was h1.
Now we have word "YMCAs" as h2. So no need for this code.